tensorflow_addons/losses/quantiles.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Implements pinball loss."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow.keras.backend as K
+import tensorflow as tf
+
+
+@tf.keras.utils.register_keras_serializable(package='Addons')
+def pinball_loss(y_true, y_pred, tau):
+    """Computes the pinball loss between `y_true` and `y_pred`.
+
```

Comment:
Would you mind including some references and more detailed description here? Maybe something like this is enough:
https://github.com/tensorflow/tensorflow/blob/r2.0/tensorflow/python/keras/losses.py#L563-L589

Reply:
ok